gen-web/src/index.js

```diff
@@ -338,11 +346,20 @@
    * @param {string} email
    */
   const validateEmail = (email) => {
-    // const re = /^\S+@\S+$/
     const re = /[^@]+@[^\.]+\..+/g
     return re.test(String(email).toLowerCase())
   }
 
+  /**
+   * Validate if string is valid size
+   * @param {string} registrationCode
+   */
+  const validateRegistrationCode = (registrationCode) => {
+    // TODO: define what the min size of the registration code will be
```

Comment:
Do we have this need/Q tracked in V1 somewhere?

Reply:
yes, I have made C and Rob aware of it